pkg/repository/provider/provider.go

```diff
@@ -21,12 +21,14 @@ import (
 	"time"
 
 	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
+	"github.com/vmware-tanzu/velero/pkg/repository/udmrepo"
 )
 
 // RepoParam includes the parameters to manipulate a backup repository
 type RepoParam struct {
-	BackupLocation *velerov1api.BackupStorageLocation
-	BackupRepo     *velerov1api.BackupRepository
+	BackupLocation          *velerov1api.BackupStorageLocation
+	BackupRepo              *velerov1api.BackupRepository
+	FullMaintenanceInterval udmrepo.FullMaintenanceIntervalOptions
```

Comment:
I wonder if this should be part of the `BackupRepositorySpec`, which is part of `BackupRepository` similarly to:
  https://github.com/vmware-tanzu/velero/blob/054375093d3e60654b2e0ec93f735e01b0083341/pkg/apis/velero/v1/backup_repository_types.go#L43

Reply:
`BackupRepository` CR is a low level API and should be manipulated by Velero internally unless really necessary.
Therefore, let's add this option to configmap only.

Reply:
FYI, `MaintenanceFrequency` of BackupRepository CR is set by Velero according to the `--default-repo-maintain-frequency` server option, unified repo interface `DefaultMaintenanceFrequency` method or `defaultMaintainFrequency` global value, so it should be edited by users directly unless really necessary.

Reply:
I was not aware `backup-repository-configmap` was a thing.. will check. Thanks.